### PR TITLE
docs(FAQ): add a note in "How to allow all IPs to access Admin API"

### PR DESCRIPTION
**Pull request by Firstsawyou**
### What this PR does / why we need it:

Signed-off-by: Firstsawyou yuelinz99@gmail.com

Add a warning note in the `How to allow all IPs to access the Admin API` section of the FAQ.

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -480,3 +480,5 @@ apisix:
 ```
 
 Restart or reload APISIX, all IPs can access the `Admin API`.
+
+>Note: You can use this method in the exercise to allow all IPs to access your `Apache APISIX` instance, but it is not safe to use it in a production environment. In production, please only authorize specific IP addresses or address ranges to access your instance.
```
Comment (spacewander) [inline docs/en/latest/FAQ.md:484]: Better to use `**...**` for Note.

Comment (Firstsawyou) [reply docs/en/latest/FAQ.md]: Updated.

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/FAQ.md [hunk 2]
```diff
@@ -479,3 +479,5 @@ apisix:
 ```
 
 重启或 reload APISIX，所有 IP 便可以访问 `Admin API`。
+
+**注意：您可以在练习中使用此方法，以允许所有 IP 访问您的 `Apache APISIX` 实例，但是在生产环境中使用它并不安全。在生产中，请仅授权特定的 IP 地址或地址范围访问您的实例。**
```
Comment (starsz) [inline docs/zh/latest/FAQ.md:483]: `练习中` is not very good.

Comment (Firstsawyou) [reply docs/zh/latest/FAQ.md]: Updated.

File: docs/en/latest/FAQ.md [hunk 3]
```diff
@@ -480,3 +480,5 @@ apisix:
 ```
 
 Restart or reload APISIX, all IPs can access the `Admin API`.
+
+**Note: You can use this method in a non-production environment to allow all IPs to access your `Apache APISIX` instance, but it is not safe to use it in a production environment. In production, please only authorize specific IP addresses or address ranges to access your instance.**
```
Comment (tokers) [inline docs/en/latest/FAQ.md:484]:
```suggestion
**Note: You can use this method in a non-production environment to allow all clients from anywhere to access your `Apache APISIX` instances, but it is not safe to use it in a production environment. In production, please only authorize specific IP addresses or address ranges to access your instance.**
```

Comment (Firstsawyou) [reply docs/en/latest/FAQ.md]: Updated.

**Review by tokers — APPROVED** [review]

**Pull request merged**